package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "reactjs_gmp_2021",
+  "version": "1.0.0",
+  "description": "ReactJS Global mentoring Program [2021Q1 IM]",
+  "main": "index.js",
+  "scripts": {
+    "start:prod": "cross-env NODE_ENV=production webpack serve --open --config webpack.config.js",
+    "start:dev": "cross-env NODE_ENV=development webpack serve --open --config webpack.config.js",
+    "build:prod": "cross-env NODE_ENV=production webpack --config webpack.config.js",
+    "build:dev": "cross-env NODE_ENV=development webpack --config webpack.config.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/KavvaP17/ReactJS_GMP_2021.git"
+  },
+  "author": "Aliaksandr Kamianchuk",
+  "license": "ISC",
+  "dependencies": {
+    "jest": "^26.6.3",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-redux": "^7.2.2",
+    "react-router": "^5.2.0",
+    "redux": "^4.0.5"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.1.5",
+    "@babel/core": "^7.1.6",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/plugin-transform-runtime": "^7.1.0",
+    "@babel/preset-env": "^7.1.6",
+    "@babel/runtime": "^7.1.5",
+    "@types/html-webpack-plugin": "^3.2.4",
+    "@types/react": "^17.0.2",
+    "@types/react-dom": "^17.0.1",
+    "@types/react-redux": "^7.1.16",
+    "@types/react-router": "^5.1.11",
+    "@types/redux": "^3.6.0",
```

Comment:
The types for redux are here, but there is no redux itself. Maybe it makes no sense to have something you don't use yet. Or you can install redux right now. Anyway you will need to have it.

Reply:
You will also need 'formik', 'yup', 'react-testing-library', 'lodash', 'jest' later on.

Reply:
'react-error-boundary'